.github/workflows/ci.yaml

```diff
@@ -140,7 +140,12 @@ jobs:
           key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
 
       - name: Public SQL Logic Tests
-        run: just slt -v 'sqllogictests/*'
+        run: |
+          if [[ "$ACTIONS_STEP_DEBUG" == "true" ]]; then
```

Comment:
I think the reason I am inclined to doing this in the slt/justfile is that, this is a bashism, and I _think_ on ubuntu I would not be surprised if this ends up being dash.